src/lab.rs

```diff
@@ -168,6 +168,23 @@ fn test_timeout(baseline_outcome: &Option<ScenarioOutcome>, options: &Options) -
     } else if options.baseline == BaselineStrategy::Skip {
         warn!("An explicit timeout is recommended when using --baseline=skip; using 300 seconds by default");
         Duration::from_secs(300)
+    } else if let Some(multiplier) = options.test_timeout_multiplier {
+        let timeout = max(
+            options.minimum_test_timeout,
```

Comment:
This function is now getting a bit nontrivial. How about adding a unit test in this file that constructs Options from Args (there are other examples of tests doing this), constructs a baseline outcome, and then checks the expected duration.